data/mock/trieStub.go

```diff
@@ -135,3 +136,11 @@ func (ts *TrieStub) AppendToOldHashes(hashes [][]byte) {
 		ts.AppendToOldHashesCalled(hashes)
 	}
 }
+
+func (ts *TrieStub) Snapshot() error {
+	if ts.SnapshotCalled != nil {
+		return ts.SnapshotCalled()
+	}
+
```

Comment:
delete empty line. In a case of stub return nil.

Reply:
Done.